.github/workflows/peribolos-schema-validate.yaml

```diff
@@ -29,7 +29,10 @@ jobs:
       with:
         fetch-depth: 0
         lfs: true
-    - run: pip install strictyaml --break-system-packages
+    - uses: actions/setup-python@v5
+      with:
+        python-version: "3.11"    
+    - run: pip install strictyaml
```

Comment:
It seems we can go with `--break-system-packages` or pin Python to a version that works without the flag. I don't know if there are any drawbacks with `--break-system-packages`, but it's one liner in a check we always can revisit.

Reply:
`pipx` doesn't work for libraries:
```
➜  community git:(pr/pipx-fix) pipx install strictyaml

No apps associated with package strictyaml or its dependencies. If you are attempting to install a library, pipx should not be used. Consider using pip or a similar tool instead.
```